src/app/statistics/angulartics/dspace/view-tracker.component.ts

```diff
@@ -36,12 +36,14 @@ export class ViewTrackerComponent implements OnInit, OnDestroy {
     this.sub = this.referrerService.getReferrer()
       .pipe(take(1))
       .subscribe((referrer: string) => {
+        let dc_identifier  = this.object.firstMetadataValue("dc.identifier.uri");
```

Comment:
What if this.object will be null?
Pls use `this.object && ...` or `this.object?.firstMe....`

Reply:
Sure, I'll add that.

Btw if `this.object` is actually null, there'll be issues down the line, eg. in the event handler https://github.com/dataquest-dev/dspace-angular/blob/654611da2db9fb77c9290b726d48dd68589116c2/src/app/statistics/statistics.service.ts#L41

Reply:
@milanmajchrak unresolved comments?